steps/browser_steps.py

```diff
@@ -23,6 +23,8 @@ def step_impl(context, url):
         if resp.status_code < 500:
             context.log.info('GET {} [{}]'.format(url, resp.status_code))
             resp.status_code.should.equal(200)
+            if context.browser.page is None:
```

Comment:
IIRC from researching this a couple weeks ago, the underlying python libraries do not include a way to get the content of the response if it doesn't look like html/xml (in other words, `context.browser.page` will be None/null), so when that happens we can save the text of the response to the context object and access it via the other new method I added